app/components/gleam-logo.ts

```diff
@@ -0,0 +1,130 @@
+import Component from '@glimmer/component';
+import { tracked } from '@glimmer/tracking';
+import { action } from '@ember/object';
+import { Rive } from '@rive-app/canvas';
+
+interface GleamLogoSignature {
+  Element: HTMLDivElement;
+  Args: {
+    class?: string;
```

Comment:
@Arpan-206 in Ember you don't need to accept DOM-level attributes like `class`/`style`, these just get passed along with `...attributes`. You only need to use `Args` for things that you'll explicitly pass in